packages/dev/core/src/DeviceInput/InputDevices/deviceSourceManager.ts

```diff
@@ -125,12 +118,13 @@ export class DeviceSourceManager implements IDisposable, IObservableManager {
 
     // Hidden Functions
     /**
-     * @param deviceSource
+     * @param deviceSource - Source to add
+     // eslint-disable-next-line tsdoc/syntax
      * @hidden
      */
-    public _addDevice(deviceSource: DeviceSource<DeviceType>): void {
+    public _addDevice(deviceSource: DeviceSourceType): void {
         if (!this._devices[deviceSource.deviceType]) {
-            this._devices[deviceSource.deviceType] = new Array<DeviceSource<DeviceType>>();
+            this._devices[deviceSource.deviceType] = new Array();

```

Comment:
Similar question, seems like we've lost type safety here.

Reply:
This should be the same case as the initial instantiation for _devices